package.json

```diff
@@ -12,7 +12,7 @@
     "eslint": "./node_modules/.bin/eslint ./src ./test",
     "performance": "node ./performance/index.js",
     "reset": "rm -rf ./node_modules yarn.lock; yarn cache clean; yarn install",
-    "test": "mocha --require babel-register ./test/esjs.spec.js ./test/**/*.spec.js"
+    "test": "mocha --require babel-register ./test/esjs.spec.js ./test/**/*.spec.js ./test/**/**/*.spec.js"
```

Comment:
Running the command manually worked fine without this (calling mocha directly). But, for whatever reason, `yarn` wasn't recursively searching test directory unless this was specified.